spyder/config/main.py

```diff
@@ -470,6 +470,7 @@
             ('appearance', APPEARANCE),
             ('lsp-server',
              {
+              'enable': True,

```

Comment:
I think we should remove this option (and its usage in this PR) because we can't really turn off the LSP completely.